Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -46,7 +59,184 @@ need to be enabled.
+
+*   Each directive may have several "attributes" (like the allowed URLs).
+
+*   Directives may build upon each other, a bit like CSS definitions (`Cascading Style Sheet`)
+    do. However, these are more meant to modify a basic rule on an earlier level,
+    and called "mutations". The relation of a "child rule" to it's "parent" is
+    also called "ancestor chain".
+
+..  _content-security-policy-terminology-policy:
+
+Applying the policy
+-------------------
+
+*   A final policy is compiled of all these directives, and then sent as a HTTP
+    response header `Content-Security-Policy: ...`.
+
+*   In TYPO3, directives can be specified via PHP syntax (within Extensions) and
+    YAML syntax (within site configuration).
+
+..  _content-security-policy-terminology-mutations:
+
+Mutations
+---------
+
+*   These rules can influence each other, this is where the concept of "mutations" come in.
+    The "policy builder" of TYPO3 applies each configured mutation, no matter where it was
+    defined.
+
+*   Because of this, each mutation (directive definition) needs a specific "mode" that can
+    instruct, how this mutation is applied: Should an existing directive be
+    `appended`, `removed` or `added` to the final policy.
+
+*   Each directive is then applied in regard to its defined mode and can list one or more
+    "sources" with the values of additional parameters of a directive. Sources are
+    web site addresses / URLs.
+
+..  _content-security-policy-terminology-nonces:
+
+Nonces
+------
+
+*   There are possible exemptions to directives for specific content created on specific
+    pages created by TYPO3 in your frontend (or backend modules). To verify, that these
+    exemptions are valid in a policy, a so-called "Nonce" (unique hash, a "**n**umber used **once**")
+    is created (details on `https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce`__).
+
+    *   TYPO3 can manage these Nonces and apply them were configured.
+    *   Nonces are retrieved from :php:`\TYPO3\CMS\Core\Security\ContentSecurityPolicy\ConsumableNonce`
+        and will be used for any directive within the scope of a single HTTP request.
+    *   More details are covered in :ref:`content-security-policy-nonce`.
+
+..  _content-security-policy-terminology-violations:
+
+Policy violations and reporting
+-------------------------------
+
+*   When a webpage with activated policies is shown in a client's browser, each HTML tag
+    violating the policy will not be interpreted by the browser.
+
+*   Depending on a configuration of a possible "Report", such violations can be submitted
+    back to the server and be evaluated there. TYPO3 provides such an interface to receive
+    and display reports in a backend module.
+
+*   Policies can be declared with "dispositions", to indicate how they are handled.
+    "Enforce" means that a policy is in effect, and "Report-Only" allows to only pretend
```

Comment:
```suggestion
    "Enforce" means that a policy is in effect, and "Report" allows to only pretend
```

DIspositions are "enforce" and "report", not "report-only". The resulting header is the "Content-Security-Policy-Report-Only".

Please change, as this can be confusing, as the disposition in a yaml file is "report".